rechnung/contract.py

```diff
@@ -23,6 +23,9 @@ def get_contracts(settings, year=None, month=None, cid_only=None, inactive=False
             (settings.contracts_dir / filename).read_text("utf-8")
         )
 
+        if not contract.get("active", False):
```

Comment:
As this seems like a newly introduced feature, wouldn't it be better to default to a contract being active, so that not all existing active contracts need to be explicitly marked as active, but instead only the new inactive contracts need to be marked as inactive?

```suggestion
        if not contract.get("active", True):
```

Reply:
Good point!